components/sup/src/event.rs

```diff
@@ -207,7 +205,7 @@ fn initialized() -> bool { NATS_MESSAGE_STREAM.try_get().is_some() }
 ///
 /// If `init_stream` has not been called already, this function will
 /// be a no-op.
-fn publish(subject: &'static Subject, mut event: impl EventMessage) {
+fn publish(subject: &'static str, mut event: impl EventMessage) {
     if let Some(stream) = NATS_MESSAGE_STREAM.try_get() {
```

Comment:
I think you can change `&'static str` to `&str`

Reply:
Will do.